### PR TITLE
Add node cli doc + some changes to cli page

### PR DESCRIPTION
**Pull request by SpectralHiss**
Here are the documentation changes as requested

Signed-off-by: Houssem El fekih <hassoum92@hotmail.com>

### CODE REVIEW

File: _docs/manage/node.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+Node shows a per node breakdown of your storage cluster.
+
+## Overview
+
+## Listing node information
+
+To view the state of your cluster simply run:
+```bash
+storageos node ls
```
Comment (oicheryl) [inline _docs/manage/node.md]: Add dollar signs before user commands.

File: _docs/manage/node.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+- Number of master copies of volumes or replicas
+- Total capacity in node
+- Percentage of capacity that is in use
+- The version of StorageOS installed
+
+
+## Inspect a node:
+
+For a more detailed overview of a node's status you can inspect it, 
+for example, let's inspect node 3
```
Comment (oicheryl) [inline _docs/manage/node.md]: Use declarative sentences. "To view detailed information such as state, port configuration, pool membership, health, version and capacity in JSON format, inspect the node:" and remove the line below.

File: _docs/manage/node.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
+---
+layout: guide
```
Comment (oicheryl) [inline _docs/manage/node.md]: You need to add this to storageos.github.io/_guides/003-manage_cluster.md.

File: _docs/manage/node.md [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+NAME                  ADDRESS             HEALTH              SCHEDULER           VOLUMES             TOTAL               USED                VERSION             LABELS
+vol-test-2gb-lon101   46.101.50.155       Healthy 2 days      true                M: 0, R: 2          77.43GiB            5.66%               0.7 (00ab7b3 rev)
+vol-test-2gb-lon102   46.101.50.231       Healthy 2 days      false               M: 1, R: 0          38.71GiB            5.90%               0.7 (00ab7b3 rev)
+vol-test-2gb-lon103   46.101.51.16        Healthy 2 days      false               M: 1, R: 1          77.43GiB            5.61%               0.7 (00ab7b3 rev)
+```
+
+From the output of the command we observe that we have a 3 node StorageOS cluster and the following information:
+- IP addresses of the controller nodes
+- The health of the cluster and uptime
+- Whether the node is a scheduler node or not
```
Comment (oicheryl) [inline _docs/manage/node.md]: Explain what a scheduler node is.

File: _docs/manage/node.md (hunk 2 above)
Comment (SpectralHiss) [inline _docs/manage/node.md]: 👍

File: _docs/manage/node.md (hunk 4 above)
Comment (SpectralHiss) [inline _docs/manage/node.md]: it's too long to explain to be honest.. according to karolis:
it's a volume scheduler
it's responsible for HA as well
identifying fallen nodes
performing healthchecks

File: _docs/manage/node.md [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+```bash
+storageos node ls
+
+NAME                  ADDRESS             HEALTH              SCHEDULER           VOLUMES             TOTAL               USED                VERSION             LABELS
+vol-test-2gb-lon101   46.101.50.155       Healthy 2 days      true                M: 0, R: 2          77.43GiB            5.66%               0.7 (00ab7b3 rev)
+vol-test-2gb-lon102   46.101.50.231       Healthy 2 days      false               M: 1, R: 0          38.71GiB            5.90%               0.7 (00ab7b3 rev)
+vol-test-2gb-lon103   46.101.51.16        Healthy 2 days      false               M: 1, R: 1          77.43GiB            5.61%               0.7 (00ab7b3 rev)
+```
+
+From the output of the command we observe that we have a 3 node StorageOS cluster and the following information:
```
Comment (oicheryl) [inline _docs/manage/node.md]: Avoid using "we", extraneous sentences and use written out numerals for numbers <10. "The output shows a three node StorageOS cluster with three nodes named vol-test-2gb-lon101, vol-test-2gb-lon102 and vol-test-2gb-lon103."

File: _docs/manage/node.md [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+- IP addresses of the controller nodes
+- The health of the cluster and uptime
+- Whether the node is a scheduler node or not
+- Number of master copies of volumes or replicas
+- Total capacity in node
+- Percentage of capacity that is in use
+- The version of StorageOS installed
```
Comment (oicheryl) [inline _docs/manage/node.md]: Avoid listing out information that's already clear from context. Think about what does a user not know. It's obvious that Health means whether it's healthy.

File: _docs/manage/node.md [hunk 7]
```diff
@@ -0,0 +1,107 @@
...
+
+NAME                  ADDRESS             HEALTH              SCHEDULER           VOLUMES             TOTAL               USED                VERSION             LABELS
+vol-test-2gb-lon101   46.101.50.155       Healthy 2 days      true                M: 0, R: 2          77.43GiB            5.66%               0.7 (00ab7b3 rev)
+vol-test-2gb-lon102   46.101.50.231       Healthy 2 days      false               M: 1, R: 0          38.71GiB            5.90%               0.7 (00ab7b3 rev)
+vol-test-2gb-lon103   46.101.51.16        Healthy 2 days      false               M: 1, R: 1          77.43GiB            5.61%               0.7 (00ab7b3 rev)
+```
+
+From the output of the command we observe that we have a 3 node StorageOS cluster and the following information:
+- IP addresses of the controller nodes
+- The health of the cluster and uptime
```
Comment (oicheryl) [inline _docs/manage/node.md]: Is this actually the health of the cluster, or of the node?

File: _docs/manage/node.md (hunk 6 above)
Comment (SpectralHiss) [inline _docs/manage/node.md]: i was going through them in corresponding order to column in output..

File: _docs/manage/node.md (hunk 4 above)
Comment (oicheryl) [inline _docs/manage/node.md]: This is why it needs to be explained. It's not obvious to a user what a scheduler does.

File: _docs/manage/node.md (hunk 5 above)
Comment (SpectralHiss) [inline _docs/manage/node.md]: agreed

File: _docs/manage/node.md (hunk 7 above)
Comment (SpectralHiss) [inline _docs/manage/node.md]: health of the controller to be precise

File: _docs/manage/node.md [hunk 8]
```diff
@@ -0,0 +1,107 @@
+---
+layout: guide
+title: StorageOS Docs - Node diagnostics
```
Comment (oicheryl) [inline _docs/manage/node.md]: I'd remove diagnostics, so it matches the titles of the other guides.

File: _docs/manage/node.md [hunk 9]
```diff
@@ -0,0 +1,107 @@
...
+
+# Node
+
+Node shows a per node breakdown of your storage cluster.
+
+## Overview
+
+## Listing node information
+
+To view the state of your cluster simply run:
```
Comment (oicheryl) [inline _docs/manage/node.md]: Remove "simply".

File: _docs/reference/cli.md [hunk 10]
```diff
@@ -52,7 +58,8 @@ Each of the storageos management commands requires a subcommand to run. Use `sto
 | Command     | Subcommands                   | Description                                                    |
 |-------------|-------------------------------|----------------------------------------------------------------|
 | `volume`    | `create inspect ls rm update` | StorageOS data volumes                                         |
-| `rule`      | `inspect ls rm update`        | Policy enforcement based on labels.                            |
+| `node`      | `ls inspect`                  | Node diagnostic information.                                   |
```
Comment (oicheryl) [inline _docs/reference/cli.md]: Remove "diagnostic".

Comment (SpectralHiss) [inline _docs/reference/cli.md]: ok

File: _guides/003-manage_cluster.md [hunk 11]
```diff
@@ -32,5 +32,8 @@ sections:
     title: Pools
     description: Create and manage pools
     module: manage/pools
-
+  -
+    title: Node
+    description: view node information
```
Comment (oicheryl) [inline _guides/003-manage_cluster.md]: Capitalize View

File: _docs/manage/node.md [hunk 12]
```diff
@@ -0,0 +1,98 @@
...
+```bash
+$ storageos node ls
+
+NAME                  ADDRESS             HEALTH              SCHEDULER           VOLUMES             TOTAL               USED                VERSION             LABELS
+vol-test-2gb-lon101   46.101.50.155       Healthy 2 days      true                M: 0, R: 2          77.43GiB            5.66%               0.7 (00ab7b3 rev)
+vol-test-2gb-lon102   46.101.50.231       Healthy 2 days      false               M: 1, R: 0          38.71GiB            5.90%               0.7 (00ab7b3 rev)
+vol-test-2gb-lon103   46.101.51.16        Healthy 2 days      false               M: 1, R: 1          77.43GiB            5.61%               0.7 (00ab7b3 rev)
+```
+
+The output shows a three node StorageOS cluster with three nodes named vol-test-2gb-lon101, vol-test-2gb-lon102 and vol-test-2gb-lon103.
```
Comment (oicheryl) [inline _docs/manage/node.md]: Remove the first "three node".

File: _docs/manage/node.md [hunk 13]
```diff
@@ -0,0 +1,98 @@
...
+For each of the nodes, details about their status is shown. Among other information, `SCHEDULER` shows if a node is a scheduler node or not.
+A scheduler node is responsible for the placement of volumes, performing health checks and providing high availability to nodes. 
+The Number of master copies of volumes or replicas is shown under `VOLUMES`.
```
Comment (oicheryl) [inline _docs/manage/node.md]: I'd restructure line 27-29 as:

* `SCHEDULER`: Whether the node contains the scheduler, which is responsible for the placement of volumes, performing health checks and providing high availability to nodes. A cluster will have exactly one scheduler node.
* `VOLUMES`: The number of master or replica copies of volumes on this node.

File: _docs/manage/node.md [hunk 14]
```diff
@@ -0,0 +1,98 @@
+---
+layout: guide
+title: StorageOS Docs - Node
+anchor: manage
+module: manage/node
+---
+
+# Node
+
+Node shows a per node breakdown of your storage cluster.
```
Comment (oicheryl) [inline _docs/manage/node.md]: Replace "your" with "the".

File: _docs/manage/node.md [hunk 15]
```diff
@@ -0,0 +1,98 @@
...
+
+## Overview
```
Comment (oicheryl) [inline _docs/manage/node.md]: Remove this line.

File: _docs/manage/node.md [hunk 16]
```diff
@@ -0,0 +1,98 @@
...
+
+## Listing node information
```
Comment (oicheryl) [inline _docs/manage/node.md]: "List node information"

File: _docs/manage/node.md (hunk 15 above)
Comment (SpectralHiss) [inline _docs/manage/node.md]: what do you mean? the whole line starting with Node?

Comment (oicheryl) [inline _docs/manage/node.md]: ## Overview. You don't have any other sections, so this is redundant.

Comment (SpectralHiss) [inline _docs/manage/node.md]: I removed this in amends

**Pull request merged**